webapp/app.py

```diff
@@ -146,6 +148,11 @@ def utility_processor():
 app.add_url_rule("/security/notices/<notice_id>", view_func=notice)
 
 
+# cve section
+app.add_url_rule("/security/cve", view_func=cve_index)
+app.add_url_rule("/security/<regex('cve-d{4}-d{4,7}'):cve_id>", view_func=cve)
```

Comment:
why not just `security/cve/<cve_id>`

Reply:
It was requested by [design]
(https://docs.google.com/document/d/1VuAElUpFay7MBFNmE3NhSyY9TkXiqHcA1ZHQ11AYO-Q/edit#heading=h.ua7qthpl074)